lib/cxx-api/include/clingo/control.hh

```diff
@@ -220,6 +283,110 @@ struct ProfileNodeInternal {
     std::vector<ProfileNode> children;
 };
 
+//! Enumeration of the results of a grounding process.
+enum class GroundResult : clingo_ground_result_t {
```

Comment:
How is this different from the `GroundResult` in `core.hh`?

Reply:
It is not different at all. It's just that the two involved libraries only have an internal dependency. The headers of the internal one are not even installed. Thus, there is a bit of duplication.